src/Tools/VMC/Utils.cxx

```diff
@@ -0,0 +1,34 @@
+#include "Tools/VMC/Utils.h"
+
+#include "Framework/Messenger/Messenger.h"
+
+#include <wordexp.h>
+
+//___________________________________________________________________________
+// REVIEW is there a built-in tool for doing this?
```

Comment:
Not exactly -- in that none use `wordexp` to do all of:
>  The expansion done consists of the following stages: tilde expansion
       (replacing ~user by user's home directory), variable substitution
       (replacing $FOO by the value of the environment variable FOO),
       command substitution (replacing $(command) or `command` by the output
       of command), arithmetic expansion, field splitting, wildcard
       expansion, quote removal.
I'm not so familiar with `wordexp` ... if wildcard expansion yields more than one file, what's the result?  And can your code handle that?

Reply:
wordexp() can handle it, but I abort in that case. L28.

Reply:
The main point here is so that experiments can specify something like $EVENT_LIBRARY_DIR/lib.root and have that managed by ups or some similar system.